src/GraphQL.Tests/Introspection/SchemaIntrospectionTests.cs

```diff
@@ -27,6 +29,53 @@ public async Task validate_core_schema(IDocumentWriter documentWriter)
             ShouldBe(json, IntrospectionResult.Data);
         }
 
+        [Theory]
+        [ClassData(typeof(DocumentWritersTestData))]
+        public async Task validate_core_schema_pascal_case(IDocumentWriter documentWriter)
+        {
+            var documentExecuter = new DocumentExecuter();
+            var executionResult = await documentExecuter.ExecuteAsync(_ =>
+            {
+                _.Schema = new Schema
+                {
+                    Query = new TestQuery(),
+                };
+                _.NameConverter = PascalCaseNameConverter.Instance;
+                _.Query = SchemaIntrospection.IntrospectionQuery;
+            });
+
+            var json = await documentWriter.WriteToStringAsync(executionResult);
+
+            ShouldBe(json, IntrospectionResult.Data);
+        }
+
+        [Theory]
+        [ClassData(typeof(DocumentWritersTestData))]
+        public async Task validate_core_schema_abstractnameconverter(IDocumentWriter documentWriter)
+        {
+            var documentExecuter = new DocumentExecuter();
+            var executionResult = await documentExecuter.ExecuteAsync(_ =>
+            {
+                _.Schema = new Schema
+                {
+                    Query = new TestQuery(),
+                };
+                _.NameConverter = new TestNameConverter();
+                _.Query = SchemaIntrospection.IntrospectionQuery;
+            });
+
+            var json = await documentWriter.WriteToStringAsync(executionResult);
+
+            ShouldBe(json, IntrospectionResult.Data);
+        }
+
+        public class TestNameConverter : INameConverter
```

Comment:
Methods of this converter are never called?

Reply:
Not explicitly.  The class is passed to `ExecutionOptions` and a `DocumentExecuter` executes the query - see test `validate_core_schema_abstractnameconverter` which is located immediately above this class definition.  The purpose of the test is to verify that given a very wacky name converter, all introspection fields still return with the proper case / spelling / etc.

Reply:
But there are no fields other than introspection fields. Right? So these methods are never called. So I propose to throw exception from rhem to explicitly show that
> given a very wacky name converter, all introspection fields still return with the proper case / spelling / etc.

Reply:
Good idea!  Done.

Reply:
I didn't understand your question at first.  Nope they are never called, and updating the test confirms it.